ReportSimulationOutput/measure.rb

```diff
@@ -1035,9 +1037,9 @@ def get_outputs(runner, args, building_id)
       batt_loss = get_report_variable_data_timeseries(keys, vars, UnitConversions.convert(1.0, 'J', 'kWh'), 0, resilience_frequency)
 
       min_soc = elcd.minimumStorageStateofChargeFraction
-      batt_kw = battery.rated_power_output / 1000.0
-      batt_roundtrip_eff = battery.round_trip_efficiency
-      batt_kwh = Battery.get_usable_capacity_kWh(battery)
+      batt_kw = elcd.designStorageControlDischargePower.get / 1000.0
+      batt_roundtrip_eff = elcs.dctoDCChargingEfficiency
+      batt_kwh = elcs.additionalProperties.getFeatureAsDouble('UsableCapacity_kWh').get
```

Comment:
For the reslience output metric, now getting values from the OpenStudio objects rather than the HPXML Battery object so I don't have to duplicate the prorating equations from battery.rb here.